CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Ruler: `-ruler.alertmanager-url` is now deprecated and has been replaced by `-ruler.alertmanager-hosts` along with `-ruler.alertmanager-path-prefix`, `-ruler.alertmanager-scheme`, `-ruler.alertmanager-username` and `-ruler.alertmanager-password`. `-ruler.alertmanager-url` will be removed in v1.4.0. #XXXX
```

Comment:
I assume these are the deprecation rules based on what I read. I assume we only need to support not erroring when you used it as opposed to supporting it in the code itself.

Reply:
Please don't break my running system now we are past 1.0

Reply:
Yep, we agreed on DMs that no breaking changes are to be made.

Reply:
Based on the conversation in #2860, this has taken a new approach.